app/dashboard/models.py

```diff
@@ -273,13 +290,13 @@ class Bounty(SuperModel):
     fulfillment_started_on = models.DateTimeField(null=True, blank=True)
     canceled_on = models.DateTimeField(null=True, blank=True)
     canceled_bounty_reason = models.TextField(default='', blank=True, verbose_name=_('Cancelation reason'))
-    project_type = models.CharField(max_length=50, choices=PROJECT_TYPES, default='traditional')
-    permission_type = models.CharField(max_length=50, choices=PERMISSION_TYPES, default='permissionless')
+    project_type = models.CharField(max_length=50, choices=PROJECT_TYPES, default='traditional', db_index=True)
+    permission_type = models.CharField(max_length=50, choices=PERMISSION_TYPES, default='permissionless', db_index=True)
     repo_type = models.CharField(max_length=50, choices=REPO_TYPES, default='public')
     snooze_warnings_for_days = models.IntegerField(default=0)
     is_featured = models.BooleanField(
         default=False, help_text=_('Whether this bounty is featured'))
-    featuring_date = models.DateTimeField(blank=True, null=True)
+    featuring_date = models.DateTimeField(blank=True, null=True, db_index=True)
```

Comment:
@owocki / @danlipert do we need to index  `featuring_date`?

Reply:
what do we need to fix about it?

> indexing == yay but only if it's legit needed right ?

 IMHO the trade off here is pretty positive ROI.  indexing on a table the size of `dashboard_bounty` is cheap (in terms of storage space + write speed) + it improves lookup times quite significantly